lib/topologies/replset.js

```diff
@@ -414,10 +414,10 @@ var pingServer = function(self, server, cb) {
 
       // Calculate latency
       var latencyMS = new Date().getTime() - start;
+
       // Set the last updatedTime
-      var hrTime = process.hrtime();
-      // Calculate the last update time
-      server.lastUpdateTime = hrTime[0] * 1000 + Math.round(hrTime[1] / 1000);
+      var hrtime = process.hrtime();
+      server.lastUpdateTime = (hrtime[0] * 1e9 + hrtime[1]) / 1e6;
```

Comment:
nit: constants for `1e9` and `1e6`

Reply:
honestly in the new code paths everything is well named (e.g. `calculateDurationInMS`), for the legacy code I'd just like to get the fix in

Reply:
I approved, the nit can be ignored.